.schema/openapi/templates/go/model_anyof.mustache

```diff
@@ -20,7 +20,7 @@ func (dst *{{classname}}) UnmarshalJSON(data []byte) error {
 	{{#-first}}
 	// use discriminator value to speed up the lookup
 	var jsonDict map[string]interface{}
-	err := json.Unmarshal(data, &jsonDict)
+	err = json.Unmarshal(data, &jsonDict)
```

Comment:
I actually don't think we need these templates any more. Instead, we should just use the templates from openapi-generator. These templates are very old and probably buggy

Reply:
done